ETC.tsv

```diff
@@ -11730,7 +11730,7 @@ ETC_20150714_011731	Underground Grave of Ritinis
 ETC_20150714_011732	Mage Tower
 ETC_20150714_011733	Viltis Forest
 ETC_20150714_011734	Glade Hillroad
-ETC_20150714_011735	$Raukime Reservoir
+ETC_20150714_011735	$Laukyme Swampy Place
```

Comment:
@imcgames Instead of "Laukyme Swampy Place" rename it to "Laukyme Swamp" - 'place' is redundant. What do you think?